src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss

```diff
@@ -83,23 +83,23 @@
   .quickAdd_container{
     text-align: center;
     .addWorkItem{
-      padding: $pad10;
+      padding: em(10);
       i,h4{
         display: block;
       }
       i{
-        margin-bottom: $margin5;
+        margin-bottom: em(5);
       }
     }
     .addWorkItemBlk{
-      padding:$pad10;
+      padding:em(10);
     }
   }
 }
 @-moz-document url-prefix() {
   .workItemQuickAdd_story{
     textarea.form-control {
-      max-height: 26px !important;
+      max-height: em(26) !important;
```

Comment:
If we didnt put !important, the styles from patternfly are not getting over written